src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private final String supplyString = "supply";
+    private final String buyString = "buy";
+    private final String resultString = "result";
```

Comment:
constants should be upper case and snake case

Reply:
This is what the check maven gives me if I do this
![image](https://user-images.githubusercontent.com/124129674/219335950-17e2b5cd-b4a1-455d-bbc9-2fa29a92cab0.png)

Reply:
Cause constant should be static too. [See here](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)